### ISSUE TITLE
Separating broadcasts from studies

### ISSUE THREAD

**Issue by Baikalcadillac** [veloce/lichobile#1498]
Right now broadcasts are included in "Study" section, which isn't intuitive. Would be nice to have separate "Broadcasts" in "Watch" submenu, along with Lichess TV.

**Comment by Randl** [comment]
Agreed! Didn't even know they are there until I saw this issue, and they are anyway hard to find under many other studies. 
https://github.com/lichess-org/lichobile/issues/1079 and https://github.com/lichess-org/lichobile/issues/1284 also relevant

**Comment by Mastimouchuk** [comment]
This is a must have! I am not able to find GCT 2023 in the Studies :(
